valuetypes/asciidoc/applib/src/main/java/org/apache/isis/valuetypes/asciidoc/applib/value/Converter.java

```diff
@@ -32,32 +32,6 @@
 
 final class Converter {
 
-    /**
```

Comment:
TODO: we should at least mention, that client side Prism needs to be involved - which is wired up to work out of the box when using the our AsciiDoc value type